test/models/default_pet_task_test.rb

```diff
@@ -0,0 +1,35 @@
+require "test_helper"
+
+class DefaultPetTaskTest < ActiveSupport::TestCase
+  should validate_presence_of(:name)
+  should validate_presence_of(:description)
+
+  test "should have valid factory" do
+    assert build(:default_pet_task).valid?
+  end
+
+  test "should belong to an organization" do
+    default_pet_task = build(:default_pet_task)
+    assert_difference("default_pet_task.organization.default_pet_tasks.count", 1) do
+      default_pet_task.save
+    end
+  end
```

Comment:
```suggestion
  should belong_to(:organization)
```
You can double check shoulda but there is a matcher for this :)

Reply:
So turns out @edwinthinks 
```
context "associations" do
    should belong_to(:organization)
  end
```
works if I use `  belongs_to :organization` on the `DefaultPetTask` model.
But if I use `acts_as_tenant(:organization)` on the model that shoulda block fails. 

This is where I wonder which is more appropriate to use as association. The OrganizationProfile uses `belongs_to` whereas Pet uses `acts_as_tenant`. Is there a 'better' approach? I see for models that use `acts_as_tenant` they do not assert the belongs_to relationship with shoulda.

Reply:
Anything that needs to be scoped to the organization tenant should use `acts_as_tenant`.